app/install.php

```diff
@@ -154,12 +154,18 @@ function saveStep2() {
 		@unlink(DATA_PATH . '/config.php');	//To avoid access-rights problems
 		file_put_contents(DATA_PATH . '/config.php', "<?php\n return " . var_export($config_array, true) . ";\n");
 
+		if (function_exists('opcache_reset')) {
+			opcache_reset();
+		}
+
 		Minz_Configuration::register('system', DATA_PATH . '/config.php', FRESHRSS_PATH . '/config.default.php');
 		FreshRSS_Context::$system_conf = Minz_Configuration::get('system');
 
 		$ok = false;
 		try {
+			Minz_Session::_param('currentUser', $config_array['default_user']);
```

Comment:
This is before we actually have the name of a real user, so use default one

Reply:
Yes, maybe by using only the Context object, and if so it would not be a big change